### DISCUSSION TITLE
Github Pages infinite loading

### DISCUSSION THREAD

**Discussion by ktodorov12** [community/community#121705] [Programming Help]
### Select Topic Area

Question

### Body

Hi, first I had problems with erorr 404 because the github pages couldn't find the links to the files, but somehow I managed to fix that (not even sure how, but it loaded) because before it was just a blank screen.

Now that I've managed to fix it, I'm stuck in infinite loading and nothing changes, it's just stuck on the home page and the loading just spins, but locally it works properly.

This is the link to the deployed version: https://ktodorov12.github.io/the-quizzes.github.io/

and this is the link to the repository: https://github.com/ktodorov12/the-quizzes.github.io

So I'd be happy if anyone has any idea why this might be happening or how to fix it.

And a small additional question: how can I remove the link to my github account in the deployed url?

Cheers! 🍺

**Comment by LiteBrite82** [comment]
Hi @ktodorov12 !

Thanks for being a part of the community!!! 

the `Pages` category is an ok place to post this; but I think you would get more community participation for this specific issue in `Programming Help`. I'll move it over there for you; good luck!!!!

**Reply by ktodorov12** [reply]
Thanks for your assistance!
